src/model/CoilCoolingDXCurveFitSpeed.cpp

```diff
@@ -0,0 +1,689 @@
+  energyInputRatioModifierFunctionofTemperatureCurve.setCoefficient1Constant(0.297145);
+  energyInputRatioModifierFunctionofTemperatureCurve.setCoefficient2x(0.0430933);
+  energyInputRatioModifierFunctionofTemperatureCurve.setCoefficient3xPOW2(-0.000748766);
+  energyInputRatioModifierFunctionofTemperatureCurve.setCoefficient4y(0.00597727);
+  energyInputRatioModifierFunctionofTemperatureCurve.setCoefficient5yPOW2(0.000482112);
+  energyInputRatioModifierFunctionofTemperatureCurve.setCoefficient6xTIMESY(-0.000956448);
+  energyInputRatioModifierFunctionofTemperatureCurve.setMinimumValueofx(17.0);
+  energyInputRatioModifierFunctionofTemperatureCurve.setMaximumValueofx(22.0);
+  energyInputRatioModifierFunctionofTemperatureCurve.setMinimumValueofy(13.0);
+  energyInputRatioModifierFunctionofTemperatureCurve.setMaximumValueofy(46.0);
+  setEnergyInputRatioModifierFunctionofTemperatureCurve(energyInputRatioModifierFunctionofTemperatureCurve);
+
+  CurveQuadratic energyInputRatioModifierFunctionofAirFlowFractionCurve(model);
+  energyInputRatioModifierFunctionofAirFlowFractionCurve.setCoefficient1Constant(1.156);
+  energyInputRatioModifierFunctionofAirFlowFractionCurve.setCoefficient2x(-0.1816);
+  energyInputRatioModifierFunctionofAirFlowFractionCurve.setCoefficient3xPOW2(0.0256);
+  energyInputRatioModifierFunctionofAirFlowFractionCurve.setMinimumValueofx(0.5);
+  energyInputRatioModifierFunctionofAirFlowFractionCurve.setMaximumValueofx(1.5);
+  setEnergyInputRatioModifierFunctionofAirFlowFractionCurve(energyInputRatioModifierFunctionofAirFlowFractionCurve);
+
+  CurveQuadratic partLoadFractionCorrelationCurve(model);
+  partLoadFractionCorrelationCurve.setCoefficient1Constant(0.75);
+  partLoadFractionCorrelationCurve.setCoefficient2x(0.25);
+  partLoadFractionCorrelationCurve.setCoefficient3xPOW2(0.0);
+  partLoadFractionCorrelationCurve.setMinimumValueofx(0.0);
+  partLoadFractionCorrelationCurve.setMaximumValueofx(1.0);
+  setPartLoadFractionCorrelationCurve(partLoadFractionCorrelationCurve);
+
+  CurveBiquadratic wasteHeatModifierFunctionofTemperatureCurve(model);
+  wasteHeatModifierFunctionofTemperatureCurve.setCoefficient1Constant(1);
+  wasteHeatModifierFunctionofTemperatureCurve.setCoefficient2x(0.0);
+  wasteHeatModifierFunctionofTemperatureCurve.setCoefficient3xPOW2(0.0);
+  wasteHeatModifierFunctionofTemperatureCurve.setCoefficient4y(0.0);
+  wasteHeatModifierFunctionofTemperatureCurve.setCoefficient5yPOW2(0.0);
+  wasteHeatModifierFunctionofTemperatureCurve.setCoefficient6xTIMESY(0.0);
+  wasteHeatModifierFunctionofTemperatureCurve.setMinimumValueofx(0.0);
+  wasteHeatModifierFunctionofTemperatureCurve.setMaximumValueofx(0.0);
+  wasteHeatModifierFunctionofTemperatureCurve.setMinimumValueofy(0.0);
+  wasteHeatModifierFunctionofTemperatureCurve.setMaximumValueofy(0.0);
+  setWasteHeatModifierFunctionofTemperatureCurve(wasteHeatModifierFunctionofTemperatureCurve);
```

Comment:
Oh I see. You've made this optional curve required and are instantiating a Biquadratic curve that is always one.

It makes sense, but I find this surprising. @kbenne thoughts please?